test/util/IdTableHelpers.cpp

```diff
@@ -98,33 +104,19 @@ IdTable createRandomlyFilledIdTable(
   auto joinColumnGeneratorView = std::views::values(joinColumnWithGenerator);
 
   // Are all the join column numbers within the max column number?
-  if (auto column = std::ranges::find_if(
-          joinColumnNumberView,
-          [&numberColumns](const size_t& num) { return num >= numberColumns; });
-      column != joinColumnNumberView.end()) {
-    throw std::runtime_error(
-        absl::StrCat("The join columns", *column, " is out of bounds."));
-  }
+  AD_CONTRACT_CHECK(std::ranges::all_of(
+      joinColumnNumberView,
+      [&numberColumns](const size_t& num) { return num < numberColumns; }));
 
   // Are there no duplicates in the join column numbers?
-  for (auto it = joinColumnNumberView.begin(); it != joinColumnNumberView.end();
-       it++) {
-    if (auto column = std::ranges::find_if(
-            it + 1, joinColumnNumberView.end(),
-            [&it](const size_t& num) { return *it == num; });
-        column != joinColumnNumberView.end()) {
-      throw std::runtime_error(
-          absl::StrCat("Join column ", *column,
-                       " got at least two generator assigned to it."));
-    }
-  }
+  std::vector<size_t> sortedJoinColumnNumbers =
+      ad_utility::transform(joinColumnNumberView, std::identity{});
+  std::ranges::sort(sortedJoinColumnNumbers, {}, {});
```

Comment:
```suggestion
  std::vector<size_t> sortedJoinColumnNumbers =
      ad_utility::transform(joinColumnNumberView, std::identity{});
  std::ranges::sort(sortedJoinColumnNumbers);
```

Reply:
Fixed in [1256dd6](https://github.com/ad-freiburg/qlever/pull/1016/commits/1256dd6a35f1b18633e8ac0e80ac30695185ba3b).